### PR TITLE
App Install Page issue

### PR DESCRIPTION
**Pull request by mudassirali007**
fixed this [issue](https://github.com/osiset/laravel-shopify/issues/1071)

### CODE REVIEW

File: src/Storage/Commands/Shop.php [hunk 1]
```diff
@@ -74,6 +74,7 @@ public function setAccessToken(ShopIdValue $shopId, AccessTokenValue $token): bo
     {
         $shop = $this->getShop($shopId);
         $shop->password = $token->toNative();
+
```
Comment (lucasmichot) [inline src/Storage/Commands/Shop.php:77]: please revert

File: src/Storage/Commands/Shop.php [hunk 2]
```diff
@@ -86,6 +87,7 @@ public function clean(ShopIdValue $shopId): bool
     {
         $shop = $this->getShop($shopId);
         $shop->password = '';
+
```
Comment (lucasmichot) [inline src/Storage/Commands/Shop.php:90]: please revert

**Review by lucasmichot — CHANGES_REQUESTED** [review]
See comments
Also can you fix https://github.com/osiset/laravel-shopify/runs/5377785831?check_suite_focus=true ?

**Comment by Kyon147** [comment]
Looks like a fix was already implemented - so closing

**Pull request closed without merging**